cluster-autoscaler/core/scale_down.go

```diff
@@ -982,9 +1025,14 @@ func (sd *ScaleDown) deleteNode(node *apiv1.Node, pods []*apiv1.Pod) status.Node
 	}
 	drainSuccessful = true
 
+	if typedErr := waitForDelayDeletion(node, sd.context.ClientSet, sd.context.AutoscalingOptions.NodeDeletionDelayTimeout); typedErr != nil {
+		return status.NodeDeleteResult{ResultType: status.NodeDeleteErrorFailedToDelete, Err: typedErr}
```

Comment:
I think this warrants a new result type (e.g. NodeDeleteErrorDelayDeletionAnnotationFailedToDisappear), the FailedToDelete one was supposed to be a failure to delete from cloud provider.

Reply:
Nevermind, discussed in person and this is ok,